boston_data_test.go

```diff
@@ -0,0 +1,531 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package stat_test
+
+import "github.com/gonum/matrix/mat64"
+
+// Boston Housing Data of Harrison and Rubinfeld (1978)
+// http://dx.doi.org/10.1016/0095-0696(78)90006-2
+// http://lib.stat.cmu.edu/datasets/boston
+// Columns are;
+//  per capita crime rate by town,
+//  proportion of non-retail business acres per town,
+//  nitric oxide concentration (parts per 10 million),
+//  weighted distances to Boston employment centres,
```

Comment:
Is the spelling of centre here from the paper? If not, please use American spelling.